<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: darkgrey;
}
.QPushButton {
  padding: 5px;
  background-color: lightgrey;
  cursor: pointer;
  text-align: center;
  border: none;
  border-radius: 5px;
}

.QPushButton:hover {
    border: 1px solid #595750;
}

.QPushButton:pressed {
  background-color: #303030;
  color: #d4d4d4;
}

.QListWidget{
	border-radius: 5px;
	border: 1px solid #595750;
	background-color: lightgrey;
}


QScrollBar:vertical {
	width: 1em;
}

QPlainTextEdit, QLineEdit{
	border-radius: 5px;
	border: 1px solid #595750;
	background-color: lightgrey;
}

QListWidget:item{
	height: 24px;
}</string>
  </property>
  <widget class="QListWidget" name="requestsList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
